addons/l10n_in_edi/models/account_move.py

```diff
@@ -61,3 +61,9 @@ def _l10n_in_edi_is_managing_invoice_negative_lines_allowed(self):
         """
         param_name = 'l10n_in_edi.manage_invoice_negative_lines'
         return bool(self.env['ir.config_parameter'].sudo().get_param(param_name))
+
+    def _can_force_cancel(self):
+        # OVERRIDE
+        """ Mark this invoice cencel(forcefully) without waiting for the EDI document to be canceled. """
```

Comment:
```suggestion
```
nitpicking: personally, I think it's not needed, but feel free to keep it (making it consistent with the inherited method) if you prefer